bin/utils/convert-argv.js

```diff
@@ -107,14 +107,17 @@ module.exports = function(...args) {
 		const requireConfig = function requireConfig(configPath) {
 			let options = (function WEBPACK_OPTIONS() {
 				if (argv.configRegister && argv.configRegister.length) {
-					module.paths.unshift(path.resolve(process.cwd(), "node_modules"), process.cwd());
+					module.paths.unshift(path.resolve(process.cwd(), "node_modules"));
 					argv.configRegister.forEach(dep => {
-						require(dep);
+						const isRelative = dep.startsWith("./") || dep.startsWith("../") || dep.startsWith(".\\") || dep.startsWith("..\\");
+						if (isRelative) {
```

Comment:
This seems to be redundant at least IMO :thinking: 

```js
const isRelative = ["./", "../", ".\", "..\\"].indexOf(dep.charAt(0)) !== -1;
```

Reply:
cc @evenstensberg

Reply:
`dep.charAt(0)` would just return '.'. And this also (more or less) the way how it's done [within node](https://github.com/nodejs/node/blob/master/lib/internal/modules/cjs/loader.js#L578).

Reply:
Yeah :sweat_smile: seems like there is no perfect way to have a similar sort of implementation to match those characters.

Reply:
Would you prefer `const isRelative = ["./", "../", ".\\", "..\\"].some(start => dep.startsWith(start));`? (even though it makes the code a bit harder to read)

Reply:
It would become more complicated as evident from the recent edit above.

Reply:
cc @evenstensberg

Reply:
> Would you prefer `const isRelative = ["./", "../", ".\\", "..\\"].some(start => dep.startsWith(start));`? (even though it makes the code a bit harder to read)

Much concise IMO :clap: